xiaomi-davinci.cfg

```diff
@@ -0,0 +1,30 @@
+[pmbootstrap]
```

Comment:
What's the point of deleting it first and then restoring it? Drop these two commits